packages/app/src/data-sync/indexes/base.indexer.ts

```diff
@@ -10,11 +10,14 @@ export abstract class BaseIndexer {
     // TODO: Will be covered with test as part of  CP-979
     return initSqlJs({
       locateFile: (fileName: string) =>
-        `${__dirname}/../static/${fileName}`,
+        // @ts-ignore
+        `${__static}/${fileName}`,
```

Comment:
to avoids ts-ignore __static could be declared, like bellow
declare const __static: string;